build.gradle

```diff
@@ -7,15 +7,17 @@ buildscript {
 
 // PLUGINS
 plugins {
-    id "java"
+    id 'java-library'
     id 'com.palantir.git-version' version "0.11.0"
 }
 
 // Apply the custom jacoco coverage plugin
 apply from: 'gradle/jacoco.coverage.gradle'
 
 group = 'org.magicdgs'
-version gitVersion()
+
+final isRelease = Boolean.getBoolean("release")
+version = (isRelease ? gitVersion() : gitVersion() + "-SNAPSHOT")
```

Comment:
If in the manifest we are changing the variable, is this still valid? Shouldn't it be archiveVersion?

Reply:
I think it gets propagated but maybe it's too confusing.  In the jar plugin they've deprecated the `jar.version ` and replaced it with `jar.archiveVersion` property which is initialized with `project.version` which is this.  I think I'll use project.version in both places.